frontend/src/svc/UserSvc.js

```diff
@@ -10,9 +10,9 @@ export const UserSvc = {
         return axios.post('/author/login', {username, password}); 
     },
 
-    Register: ({username, github_url, password}) => {
-        console.log('Register values of form: ', username, password, github_url);
-        return axios.post('/author/register', {username, password, github_url});
+    Register: ({username, git_url, password}) => {
```

Comment:
just an fyi this will be changing in the near future as we move to using actions in react, but for now its good